reana_commons/email.py

```diff
@@ -46,7 +48,7 @@ def send_email(
         if os.getenv("FLASK_ENV") != "development":
             server.starttls(context=context)
             server.login(login_email, REANA_EMAIL_PASSWORD)
-        server.sendmail(sender_email, receiver_email, message)
+        server.sendmail(sender_email, receiver_email, message.as_bytes())
```

Comment:
Do we need to specify `as_bytes(policy=email.policy.SMTP)` to be fully compliant? https://docs.python.org/3/library/email.policy.html#email.policy.SMTP

Reply:
The [`policy.SMTP` policy](https://docs.python.org/3/library/email.policy.html#email.policy.SMTP) is the same as the default one, except that it uses `\r\n` as line separator, but [`sendmail`](https://docs.python.org/3/library/smtplib.html#smtplib.SMTP.sendmail) converts `\n` to `\r\n` anyway, so this shouldn't be an issue.

In any case, I just realized that there is also [`send_message`](https://docs.python.org/3/library/smtplib.html#smtplib.SMTP.send_message) which should handle all these nuances for us.